src/pages/[...slug].astro

```diff
@@ -1,11 +1,18 @@
 ---
-import aposPageFetch from '@apostrophecms/astro-integration/lib/aposPageFetch';
-import AposLayout from '@apostrophecms/astro-integration/components/layouts/AposLayout.astro';
-import AposTemplate from '@apostrophecms/astro-integration/components/AposTemplate.astro';
+import aposPageFetch from '@apostrophecms/apostrophe-astro/lib/aposPageFetch.js';
+import AposLayout from '@apostrophecms/apostrophe-astro/components/layouts/AposLayout.astro';
+import AposTemplate from '@apostrophecms/apostrophe-astro/components/AposTemplate.astro';
 
 const aposData = await aposPageFetch(Astro.request);
 const bodyClass = `myclass`;
---- 
+
+if (aposData.redirect) {
+  return Astro.redirect(aposData.url, aposData.status);
+}
+if (aposData.notFound) {
+  Astro.response.status = 404;
```

Comment:
If we wait until where it was formerly done it is too late and does not take effect.